M:EE on an Azure VM (#5)
e3tran/MinecraftChatbot: project1/project1.md

* https://www.minecraft.net/content/dam/minecraft/addons/alien-invasion.mcworld

Don't click on the file yet! Go to the main menu of Minecraft: Education Edition and click this button: 
![Alt text](images/ImportButton.png?raw=true "Title")

Then, select the file you just downloaded. The world should be imported, but you want to make a few edits first. Make sure to click on the pencil next to your world to edit it. Set these few changes:
* Default Game Mode: Creative
* Difficulty: Hard
* Activate Cheats

You should now be ready to enter your world! Go ahead and hit "Play".

## Part 4: Create a Guide to Survive the Alien Apocalypse
First thing you may ask yourself is: what is this place?

Well, you're in the middle of an alien warzone! There are aliens and UFOs everywhere!

There are a few chests in front of you with some equipment, but that may not be sufficient if we want to save the world. What can we do?

If you did the steps above correctly, you should be in creative mode (this means that you are able to grab whatever items you want from your inventory). 

> The idea is to take whatever items you feel like will make you more powerful, and place them into your inventory. Then, once you switch back to Survival mode, you will have a much easier time fighting aliens. 

Author: ```suggestion
The idea is to take whatever items you feel like will make you more powerful, and place them into your inventory. Then, build your protective lair. You might want to add in some MakeCode scripts to create traps, or escape pods. Then, once you switch back to Survival mode, you will have a much easier time fighting aliens. 

You will then create one MakeCode script that will force anyone joining your world to go into Survival mode after 5 minutes. The challenge: get ready to survive....then survive!
```

Reviewer: that's epic :D

> Your job is to create a guide for another alien warrior to follow so that they can equip themselves with sufficient weapons to fight the aliens. Make sure to create the guide on any software that you can save as a file (for example, Microsoft Word). 

Author: ```suggestion
Your job is to create a world for another alien warrior to survive in. You should also add signs in the world to guide them through choosing the right inventory and learning how to navigate your world. 
```

Here is an example guide:
* First, open your inventory with "e". Then, switch to the weapons category and add a full set of diamond armor to your inventory. Pick out a diamond sword and shield as well, and ...

> Once you've finished, save your file!

Author: ```suggestion
Once you've finished, host your world and invite friends to join and see who can survive the longest!
```
